src/main/java/org/fogbowcloud/app/utils/CustomFileAppender.java

```diff
@@ -5,6 +5,7 @@
 import java.text.SimpleDateFormat;
 import java.util.Date;
 
+/** Utility class for Swagger configuration. */
```

Comment:
Not used for Swagger, but for log4j.

Reply:
Solved in 41d40be35da627da99f10f2df4ac6fc8c95bcb09.